src/Core/Services/Implementations/UserService.cs

```diff
@@ -554,6 +565,32 @@ public async Task InitiateEmailChangeAsync(User user, string newEmail)
             user.EmailVerified = true;
             user.RevisionDate = user.AccountRevisionDate = DateTime.UtcNow;
             await _userRepository.ReplaceAsync(user);
+
+            if (user.Gateway == GatewayType.Stripe)
+            {
+
+                try
+                {
+                    await _stripeSyncService.UpdateCustomerEmailAddress(user.GatewayCustomerId,
+                        "");
```

Comment:
Probably a mishap from testing the guards, but we should pass the user's email to the method.